### PR TITLE
first run

### PR DESCRIPTION
**Pull request by jasmingacic**

### CODE REVIEW

**Comment by jasmingacic** [comment]
Closes https://github.com/calyptia/core-operator/issues/85

File: README.md [hunk 1]
```diff
@@ -1,2 +1,100 @@
...
+
+Core instance is a set of daemons running in your cluster which have several functions: 
+* Registers itself with Cloud API 
+* Fetches Pipelines created in the UI and syncs them with the cluster. Effectively creating or deleting Pipeline resources in the cluster
+* Updates Pipelines status in the Cloud API 
+
+
+## Installation options
+
+### Calyptia CLI
```
Comment (jasmingacic) [inline README.md:7]: @gabrielbussolo please add anything i have missed here

**Comment by niedbalski** [comment]
@jasmingacic you need to configure a proper TOC and a gitbook configuration (https://docs.gitbook.com/product-tour/git-sync/content-configuration)

**Comment by jasmingacic** [comment]
https://app.gitbook.com/o/-MUQ1eTVMK9HnM2ck877/s/kPsgxXyYBUysSnkjh7oZ/~/revisions/1NVJchpZbMYZOt688qNq/

**Review by niedbalski — APPROVED** [review]

**Pull request merged**